features/harmful-apis/main.js

```diff
@@ -220,6 +226,12 @@ const tests = [
         run: () => {
             return ('joinAdInterestGroup' in navigator) || ('runAdAuction' in navigator);
         }
+    },
+    {
+        id: 'window-placement-api',
+        run: () => {
+            return ('screen' in window && 'isExtended' in window.screen);
```

Comment:
is this the multi-screen windows API? [Google](https://sheets.googleapis.com/v4/spreadsheets/1ndxh6sl0fSOLHFlMdSiLPqmGPMDVJqkJFYnQ0Hsmhwo/values/Sheet2?alt=json&key=AIzaSyCkROWBarEOJ9hQJggyrlUFulOFA4h6AW0) folks are using a different detection technique:

>       "'Multi-Screen Window Placement': {regEx: /getScreenDetails\\s*\\(\\)/g, where: 'JavaScript', supported: (async () =\u003e 'getScreenDetails' in self)(), featureDetection: `(async () =\u003e 'getScreenDetails' in self)()`, documentation: 'https://developer.chrome.com/articles/multi-screen-window-placement/', blinkFeatureID: 3388},"

Not sure if that's better?

Reply:
Both `getScreenDetails` and `isExtended` are introduced in the spec, so I guess it doesn't really matter?

Reply:
Yeah, I guess their goal is to check if feature is available FULLY. We may not care about that, for us if feature is available even partially it's alrady bad.